cluster/local/integration_tests_no_tls.sh

```diff
@@ -265,10 +266,10 @@ echo "${INSTALL_YAML}" | "${KUBECTL}" delete -f -
 timeout=60
 current=0
 step=3
-while [[ $(kubectl get providerrevision.pkg.crossplane.io -o name | wc -l) != "0" ]]; do
+while [[ $(kubectl get providerrevision.pkg.crossplane.io -o name | wc -l | tr -d '[:space:]') != "0" ]]; do
   echo "waiting for provider to be deleted for another $step seconds"
-  current=$current+$step
-  if ! [[ $timeout > $current ]]; then
+  current=$((current + step))
+  if [[ $current -ge $timeout ]]; then
```

Comment:
This was also not ever timing out (only on macOS?), because doing integer operations/comparisons requires this special syntax.